Adds value for setting the dashboard pods resources

byronmansfield opened the pull request:
I added the ability to set the resources for the dashboard pod, plus updated README helm-docs for it

byronmansfield commented:
Hi. Notes to reviewers. I realize that there are some changes greater than the original scope of work. My editor automatically trims the trailing whitespace on save. If that is undesirable, I can correct that with my PR. Additionally, I ran helm-docs to include my new values, but it seems to have caught a few additional ones that had not been updated. If this is also undesirable, I am happy to only include my addition. Just let me know. Thank you.

gianlucam76 commented:
Thank you @byronmansfield 

We need to change [this](https://github.com/projectsveltos/helm-charts/blob/64a1f67fc03a33887e7db139dc3bac6e74db816d/charts/dashboard/Chart.yaml#L17) as well (just increase patch version)

byronmansfield commented:
Great catch. I bumped the patch version and reflected that in the readme as well

gianlucam76 commented:
Thank you so much @byronmansfield 

Might I ask to please rebase and push again (just changing the chart release version to v0.55.1)? I did a new release and wanted this to come after the release (so folks can see the single change and not part of the release itself which could have been confusing)

As soon as you are done we can merge this. Thank you again!

Commit:
Adds ability to configure resources for dashboard pod

byronmansfield commented:
Hey @gianlucam76. No problem. I rebased and updated my branch. I did run a helm-docs, and the changes you see is what it produced. If you want me to leave this out, then let me know.

gianlucam76 reviewed (approved):
LGTM. Thank you
